src/redux/actions/genes/loadGeneExpression.js

```diff
@@ -7,20 +7,32 @@ import {
 
 import fetchWork from 'utils/work/fetchWork';
 import getTimeoutForWorkerTask from 'utils/getTimeoutForWorkerTask';
+import { findLoadedGenes } from 'utils/genes';
 
 const loadGeneExpression = (
-  experimentId, genes, componentUuid, useDownsampledExpression = false,
```

Comment:
Removed all the downsample handling from this file, just kept the previous parts